templates/repo/diff/box.tmpl

```diff
@@ -103,6 +103,11 @@
 									{{if $file.IsVendored}}
 										<span class="ui label gt-ml-3">{{$.locale.Tr "repo.diff.vendored"}}</span>
 									{{end}}
+									{{if and $file.Mode $file.OldMode}}
+										<span class="ui gt-ml-5 gt-mono">{{$file.Mode}} → {{$file.OldMode}}</span>
+									{{else if $file.Mode}}
+										<span class="ui gt-ml-5 gt-mono">new mode → {{$file.Mode}}</span>
```

Comment:
Same here.
In addition, the `new mode → %[1]s` should be extracted into a translatable string.

Reply:
Done